bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -258,6 +266,9 @@ public BigtableSession(BigtableOptions opts) throws IOException {
     BigtableClientMetrics.counter(MetricLevel.Info, "sessions.active").inc();
 
     // Defer the creation of both the tableAdminClient until we need them.
+    //TODO(rahulkql): Need to identify which resources to initialized for GCJ's adapter case.
```

Comment:
Need to identify which resource to initialize for GCJ's adapter case. 
or Need to identify which resources are to be initialized for GCJ's adapter case.